Extension/src/LanguageServer/client.ts

```diff
@@ -983,6 +1069,22 @@ export class DefaultClient implements Client {
                     } else {
                         this.configuration.CompilerDefaults = compilerDefaults;
```

Comment:
I'm seeing compilerDefaults be undefined here with a multi-root workspace, which leads to an undefined dereference in the assignment. The code that assigns to that got moved below.